.vib/wildfly/goss/vars.yaml

```diff
@@ -0,0 +1,20 @@
+binaries:
+  - java
+directories:
+  - mode: "0775"
+    paths:
+      - /opt/bitnami/wildfly
+      - /home/wildfly
+      - /opt/bitnami/wildfly/domain
+      - /opt/bitnami/wildfly/standalone
+      - /bitnami/wildfly/standalone/data
+  - paths:
+      - /opt/bitnami/wildfly/modules/system/layers/base
```

Comment:
Are we creating this directory in our compilation or postunpack logic? I couldn't find where are we interacting with it.

Reply:
True. In our compilation we are using it here:
```
  get trivySkipDirs() {
    return ['modules/system/layers/base'];
  }
```
But we are already checking it here:
```
  /opt/bitnami/wildfly/modules/system/layers/base/org/wildfly/openssl/main/lib/macosx-x86_64/libwfssl.dylib:
    exists: false
```
So it isn't necessary.